samples/openthread/coap_client/README.rst

```diff
@@ -46,6 +49,14 @@ Button 2:
   Pressing results in sending a multicast LIGHT_ON or a multicast LIGHT_OFF message (alternatively) to the ``/light`` resource on server nodes.
   Sending this kind of message instead of a LIGHT_TOGGLE one allows to synchronize the state of the LEDs on several server nodes at once.
 
+Button 3:
+  Pressing results in switching power consumption modes: from SED to MED and from MED to SED.
```

Comment:
I think the note below is not needed. This could be rephrased as:
```suggestion
  When the device is working as Minimal Thread Device (MTD), pressing results in switching the power consumption modes from SED to MED or from MED to SED.
```
And then under `working as Minimal Thread Device (MTD)` you can link to the user guide section about switching device types (note: you need to add a section tag there for easier linking).